app/views/profile/edit.html.erb

```diff
@@ -95,7 +95,8 @@
 
     <!-- 🌟 Courses -->
     <div class="col">
-      <div class="card profile-card" id="courses-accordion" data-cy="courses-accordion">
+      <div class="card profile-card" id="courses-accordion"
+           data-cy="courses-accordion" data-testid="courses-accordion">
```

Comment:
Did you leave the  `data-cy` attribute in because we still need it for other cypress tests to run?